doc/user_guide/metadata_structure.rst

```diff
@@ -70,7 +70,6 @@ in the following sections of this chapter.
         │   │   └── parameters_estimation_method
         │   └── variance
         ├── binned
-        ├── record_by
         ├── signal_type
         └── signal_origin
```

Comment:
I thought on leaving the metadata as the information is useful. We just don't take it into account when assigning the signal subclass.